internal/server/data/selectors.go

```diff
@@ -51,6 +51,9 @@ func ByOptionalIDs(ids []uid.ID) SelectorFunc {
 }
 
 func ByOrgID(orgID uid.ID) SelectorFunc {
+	if orgID == 0 {
```

Comment:
Looks like we don't have any protection for `add` and `save` anymore if we only rely on this selector.